From 6101b8ca63c910448daa0d76374f16c68bd9fe81 Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Mon, 4 Nov 2024 11:39:11 +0100
Subject: [PATCH 702/841] fix(stm32mp2-fdts): move stm32mp23 dk board on new
 SDMMC1 slew rate

Move stm32mp23 dk board on new SDMMC1 slew rate.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git ef328d2b8e9179a8291ffadaed1f17bfe2c93789 fix(stm32mp2-fdts): move stm32mp23 dk board on new SDMMC1 slew rate]
Change-Id: I22a75a8055182e9d37dca10ad9246b6e81d614be
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/416196
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
---
 fdts/stm32mp235f-dk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp235f-dk.dts b/fdts/stm32mp235f-dk.dts
index 775de4e9e..8953a575a 100644
--- a/fdts/stm32mp235f-dk.dts
+++ b/fdts/stm32mp235f-dk.dts
@@ -199,7 +199,7 @@
 
 &sdmmc1 {
 	pinctrl-names = "default";
-	pinctrl-0 = <&sdmmc1_b4_pins_a>;
+	pinctrl-0 = <&sdmmc1_b4_pins_b>;
 	st,neg-edge;
 	bus-width = <4>;
 	vmmc-supply = <&vdd_sdcard>;
-- 
2.47.2

